.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,8 @@
-- [ ] Tests written and linted [ℹ︎]()
-- [ ] Documentation written [ℹ︎]()
-- [ ] Commit history is tidy [ℹ︎]()
+- [ ] Tests written and linted
+- [ ] Documentation written
```

Comment:
Can we maybe say "Updated README if necessary" rather? Especially if it's an external contributor the "Documentation written" seems confusing.